src/plugin.ts

```diff
@@ -19,7 +19,7 @@ import {
   createComponentExtension,
   createPlugin,
   createRoutableExtension,
-  discoveryApiRef,
+  discoveryApiRef, identityApiRef,
```

Comment:
weird that this didn't get pushed to the next line, do you have prettier properly configured?